### ISSUE TITLE
4. Tweak the main page

### ISSUE THREAD

**Issue by edakturk14** [BuidlGuidl/batch8.buidlguidl.com#7]
## Goal

Discuss with your peers what you would like the homepage to show! It can be a good opportunity to tweak the app colors/layout etc.

As in #9, keep it simple (no big images, external assets, etc)

This can be an ongoing issue while you all keep implementing things.

**Comment by indralukmana** [comment]
Hi everyone, I would like to propose three tweaks, two tweaks on the home page itself, and one on the nav (this would be a bit more involved).

1. remove the existing 2 boxes for chain explorations information
1. add the builders list of avatars/initials and link them to the pages created in the issue #9 by individual builders
1. add builders page list and put it on the navigation, this would involve: creating a page of builders list (and linking them to each existing one), then adjust the navigation bar to link to this page

The two items on the home page might be able to be done in one issue/PR, and the builders list page would be separate issue/PR.

This would be the rough design of the points above:

![Screenshot 2024-08-13 133947](https://github.com/user-attachments/assets/37583104-e16a-43af-bec7-754a139ef5c8)

I would appreciate any comment and if they are good, I would be happy to make issues for them that anyone can take and if it is okay I would like to take a stab at creating the page and adjusting the nav 😁

**Comment by indralukmana** [comment]
Oops my bad just saw the issue #4 this should be part of the point I mentioned above, and I believe @rohan-ahire06 is working on the issue?

**Comment by rohan-ahire06** [comment]
Yes, I’m actively working on this issue at the moment. I also have a suggestion regarding the homepage that could add a fun and interactive element.

## Vending Machine Concept

### Proposal:
How about we introduce a virtual vending machine on the homepage? The idea is simple: users can make bids, and whoever places the highest bid will have their account featured on the homepage. This feature would then link directly to the builder’s detail page.

Not only does this add an exciting, gamified aspect to the site, but it also gives users an opportunity to showcase their work. Plus, you can even feature yourself on the homepage—it’s a win-win and a lot of fun!

**Comment by RafaelCaso** [comment]
@indralukmana
1. Agreed
2. Do we want all the builders immediately on the homepage? I'm just worried it will look cluttered
3. Agreed. NavBar item labeled 'Builders' that links to a list like you mentioned in item 2? Then, the avatars/initials would be on this page and would each link to the individual's profile page?

@rohan-ahire06 
Love the idea of a gamified aspect for the site. I'm not sure I fully understand the bidding concept though. When I saw 'vending machine' I thought maybe a button that randomly selects a builder and provides their avatar and links (like those kids toy dispensers at the grocery store that give a random toy in a little ball). Would love to hear more about your proposed idea

**Comment by indralukmana** [comment]
@rohan-ahire06 

That's a cool idea! Adding a gamified aspect to the project sounds fun. However, I'm a bit hesitant about implementing an extensive smart contract for this, especially since we're working on the OP mainnet. A bidding system with real money might not be something everyone wants to engage with 😆. 

Maybe we can achieve the gamified feature aspect using a more benign approach. My idea is to combine on-chain and off-chain data, which could then be weighted into a _scoring system_. This could include:

- **Off-chain**: GitHub activity (comments, discussions, PRs, emoji reactions, etc.)
- **On-chain**: Smart contract check-ins
- **On-chain**: Wallet history
- **Off-chain**: Telegram channel activity
- **Other**: Any other data that could be useful

We can compile this data and feed it into a _scoring system_. Based on your proposal, we could create a leaderboard, and those who rank highly could be featured on the homepage.

![image](https://github.com/user-attachments/assets/d5805bcc-ac9c-4cc5-a8ba-0e7483e6eb45)


The pro is that this would foster engagement and activity across the GitHub repo and beyond. The con is that I’m not sure how simple or complex this would be to implement—some parts might not be feasible due to API constraints or might require hacks or leveraging undocumented APIs.

**Comment by indralukmana** [comment]
@RafaelCaso 

For point 3 already worked on by @rohan-ahire06 in issue #4. For point 2 depends on how many builders that has checkedin and submit their PR for the personal page, around 20 persons should be okay, what we are showing should not be too much detail there, can be just small blocky avatar + short wallet address of builders, a more detailed one can be done in the builder list and the individual detail in each builder personal page.

**Comment by phipsae** [comment]
FYI: There are around 12 persons in the batch.
